src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/RenderInputParams/InputComponent/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import Col from '../../../../../../../../../components/layout/Col'
+import Field from '../../../../../../../../../components/forms/Field'
+import Checkbox from '../../../../../../../../../components/forms/Checkbox'
+import TextField from '../../../../../../../../../components/forms/TextField'
```

Comment:
why this kind of import? Can't we use absolute paths? `src/components`.

Reply:
`TextFiled` is already implemented in safe-react-components, why are we using this one?

Reply:
I tried to replace it but it can't be done easily, will require some extra effort, I would suggest doing that in another pr